src/banks/cba_au/parse-segments.ts

```diff
@@ -217,12 +217,12 @@ function stringToMoney(moneyStringWithCrDb: string): Money {
     mode = 'CR'
   }
   const [moneyString] = moneyStringWithCrDbClean.split(' ')
-  const [dollars_str, cents_str] = moneyString.replaceAll(',', '').split('.')
+  const dollars_str = moneyString.replaceAll(',', '')
 
   if (mode) {
-    return [parseInt(dollars_str, 10), parseInt(cents_str, 10), mode]
+    return [parseFloat(dollars_str), mode]
   } else {
-    return [parseInt(dollars_str, 10), parseInt(cents_str, 10), null]
+    return [parseFloat(dollars_str), null]

```

Comment:
I was hesitant to use JavaScript floats because the numbers might be subtly inaccurate. 

Perhaps a better approach would be to remove the `.`, parse as int and consider the whole value as "cents"

For example, `1.01` would be `101` cents 🤔

Reply:
Yeah I 100% agree with that logic. I had a similar thought but figured that it wasn't doing any calculations on the values so should be ok.
But wouldn't be a bad improvement if time permits :) 

Thanks for merging the changes!